crates/libs/windows/src/core/bindings.rs

```diff
@@ -1705,31 +1726,44 @@ impl ::core::ops::Not for FORMAT_MESSAGE_OPTIONS {
     }
 }
 #[inline]
-pub unsafe fn FormatMessageW<'a, Param0: ::std::convert::Into<FORMAT_MESSAGE_OPTIONS>>(dwflags: Param0, lpsource: *const ::core::ffi::c_void, dwmessageid: u32, dwlanguageid: u32, lpbuffer: ::windows::core::PWSTR, nsize: u32, arguments: *const *const i8) -> u32 {
+pub unsafe fn FormatMessageW<'a, P0>(dwflags: P0, lpsource: *const ::core::ffi::c_void, dwmessageid: u32, dwlanguageid: u32, lpbuffer: ::windows::core::PWSTR, nsize: u32, arguments: *const *const i8) -> u32
+where
+    P0: ::std::convert::Into<FORMAT_MESSAGE_OPTIONS>,
```

Comment:
Enums shouldn't be convertible...

Reply:
This is not an issue with this PR, but we should make an issue for this.

Reply:
Sure, happy either way. Just noticed it here first now that the code is so much more readable. 😆

Reply:
I'd rather do this as a follow up PR if you don't mind.